hedera-node/hedera-app/src/main/java/com/hedera/node/app/Hedera.java

```diff
@@ -602,16 +599,22 @@ public void onStateInitialized(
         } else {
             deserializedVersion = new HederaSoftwareVersion(SemanticVersion.DEFAULT, SemanticVersion.DEFAULT);
         }
-        logger.info("Deserialized version is {}, version {}", deserializedVersion, version);
+
+        // This is the *FIRST* time in the initialization sequence that we have access to the platform. Grab it!
+        // This instance should never change on us, once it has been set
+        assert this.platform == null || this.platform == platform : "Platform should never change once set";
+        this.platform = platform;
```

Comment:
Is this needed? I can see that on lines 576-579 we have the exact block of code. Maybe there was a merge conflict? Or something from the parent branch you branched off? Are the changes from this class and `ServicesMain.java` needed?

Reply:
this file is from the branch

Reply:
yeah on second look I don't think these two files (`Hedera.java` and `ServicesMain.java`) are really part of this PR.  Right?

Reply:
Fixed